package.json

```diff
@@ -126,11 +126,13 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-storybook": "^0.5.5",
     "execa": "^5.1.1",
+    "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
     "json-loader": "^0.5.7",
     "jsontosass": "^0.2.0",
     "lerna-changelog": "^2.2.0",
     "mini-css-extract-plugin": "^2.4.5",
+    "npm": "^8.12.1",
```

Comment:
why?

Reply:
Orr did it so jest can detect the classNames without the css modules.